### ISSUE TITLE
SMP: "Import" flow added to new site button

### ISSUE THREAD

**Issue by p-jackson** [Automattic/wp-calypso#67534]
Depends on: https://github.com/Automattic/wp-calypso/issues/67646.

The "Add new site" button in the SMP should also provide easy access to the "import" flow.

This is an early design
<img width="374" src="https://user-images.githubusercontent.com/1500769/189042893-a615a69c-3e8f-45c4-ad85-3ff44e2424a8.png">

The above gives a good idea of what we're going for, however design/wording may change. pdKhl6-HA-p2#comment-689

Note that Calypso has a split button component: https://wpcalypso.wordpress.com/devdocs/design/split-button
There doesn't appear to be an existing Gutenberg split button component.

It may be useful to implement this feature in multiple PRs, in which case it might be useful to create a feature flag e.g. if you introduce the split button style in its own PR without any extra actions then you can hide it behind a `build/add-new-site-split-button` flag.

Done is:
- [x] "Add new site" button uses a split button style
- [x] The default click action continues to create a new simple site (i.e. `/start` flow)
- [x] Can enter the Jetpack connect flow via a dropdown action (i.e. `/jetpack/connect` flow)
- [x] There is an `Import an existing site` option in the split button dropdown menu.
- [x] Can enter the site import flow via that dropdown action (reuse importing flows pdKhl6-FS-p2#comment-708)
- [x] It's possible using tracks data to see that the origin of an "import" was the SMP
- [ ] Still to confirm: the new site button in the `<NoSitesMessage>` has the same split button design

**Comment by sejas** [comment]
I've updated the title and a bit of the description to make reduce the scope of this issue removing the jetpack connect button and focusing only on the `Import` flow.

- Marko has created this new issue:  https://github.com/Automattic/wp-calypso/issues/67646
- There is a PR in progress with the split button and the jetpack connect link: https://github.com/Automattic/wp-calypso/pull/67662

**Comment by sejas** [comment]
@wojtekn the split button has been merged to production. Now you should be able to add a new "child" button: https://github.com/Automattic/wp-calypso/issues/67646

**Comment by wojtekn** [comment]
I've opened [a PR that adds import flow to the button](https://github.com/Automattic/wp-calypso/pull/67847).

> Still to confirm: the new site button in the <NoSitesMessage> has the same split button design

I haven't included this requirement in my PR as it was not confirmed yet. I'm not sure if we want to duplicate the dropdown below, especially since another button shows a different CTA that asks to perform a more specific action:

![Screen Shot 2022-09-15 at 10 27 29](https://user-images.githubusercontent.com/727413/190354777-51b90c7c-2d80-49fb-ae68-1a96128fe4e6.png)

If we decide to keep this button in sync, we should open another ticket.
